interpreter/python/python.go

```diff
@@ -736,9 +743,9 @@ func Loader(ebpf interpreter.EbpfHandler, info *interpreter.LoaderInfo) (interpr
 	}
 
 	// Calls first: PyThread_tss_get(autoTSSKey)
-	autoTLSKey = decodeStub(ef, pyruntimeAddr, "PyGILState_GetThisThreadState", 0)
+	autoTLSKey, err = decodeStub(ef, pyruntimeAddr, "PyGILState_GetThisThreadState")
 	if autoTLSKey == libpf.SymbolValueInvalid {
-		return nil, errors.New("unable to resolve autoTLSKey")
+		return nil, fmt.Errorf("unable to resolve autoTLSKey %w", err)
```

Comment:
As errors returned by `decodeStub()` not part of an API or documented in a proper way, we should not make them part of the API of `Loader()`.
```suggestion
		return nil, fmt.Errorf("unable to resolve autoTLSKey %v", err)
```